### PR TITLE
[TW-1591] Doc Read path of request including Collections and folders

### PR DESCRIPTION
**Pull request by mark-anthony-postman**
Adds new section to the Javascript reference describing the new `pm.execution.location` and `pm.execution.location.current` properties.

View the beta docs here: [Get the path and name of a request](https://tw-1591-read-request-path.learning.postman-beta.com/docs/writing-scripts/script-references/postman-sandbox-api-reference/#get-the-path-and-name-of-a-request)

### CODE REVIEW

File: src/pages/docs/writing-scripts/script-references/postman-sandbox-api-reference.md [hunk 1]
```diff
@@ -622,6 +623,24 @@ pm.sendRequest('https://postman-echo.com/get', (error, response) => {
 
 See the [Request definition](http://www.postmanlabs.com/postman-collection/Request.html#~definition) and [Response structure](http://www.postmanlabs.com/postman-collection/Response.html) reference docs for more detail.
 
+### Get the path and name of a request
+
+The `pm.execution.location` property enables you to get the complete path of a request, including the folder and collection, in array format. For example, for a request name **R1** in folder **F1** in collection **C1**, the following test script code will return `["C1", "F1", "R1"]`:
```
[the remark this answers is not in the document]
Comment (mark-anthony-postman) [reply src/pages/docs/writing-scripts/script-references/postman-sandbox-api-reference.md:628]: Thanks for catching that!

**Review by jkonrath-postman — APPROVED** [review]
LGTM

File: src/pages/docs/writing-scripts/script-references/postman-sandbox-api-reference.md [hunk 2]
```diff
@@ -622,6 +623,24 @@ pm.sendRequest('https://postman-echo.com/get', (error, response) => {
...
 See the [Request definition](http://www.postmanlabs.com/postman-collection/Request.html#~definition) and [Response structure](http://www.postmanlabs.com/postman-collection/Response.html) reference docs for more detail.
 
+### Get the path and name of a request
+
+The `pm.execution.location` property enables you to get the complete path of a request, including the folder and collection, in array format. For example, for a request named **R1** in folder **F1** in collection **C1**, the following test script code will return `["C1", "F1", "R1"]`:
+
+```js
+// Returns the full path of a request in array format
+console.log(pm.execution.location);
+```
```
Comment (giridharvc7) [inline src/pages/docs/writing-scripts/script-references/postman-sandbox-api-reference.md:633]: We should also add what the output of this looks like.

File: src/pages/docs/writing-scripts/script-references/postman-sandbox-api-reference.md [hunk 3]
```diff
@@ -622,6 +623,24 @@ pm.sendRequest('https://postman-echo.com/get', (error, response) => {
...
+
+To get the name of the current element, you can use the `pm.execution.location.current` property. For example, if you add the following code to the pre-request script of a folder named **F1**, it will return `F1`:
+
+```js
+// Returns the name of the current element
+console.log(pm.execution.location.current);
+```
```
Comment (giridharvc7) [inline src/pages/docs/writing-scripts/script-references/postman-sandbox-api-reference.md:640]: Same here, may be we can add what the output looks like for quick understanding

**Review by giridharvc7 — CHANGES_REQUESTED** [review]

**Review by giridharvc7 — APPROVED** [review]

File: src/pages/docs/writing-scripts/script-references/postman-sandbox-api-reference.md (hunk 2 above)
Comment (mark-anthony-postman) [reply src/pages/docs/writing-scripts/script-references/postman-sandbox-api-reference.md:634]: I changed this to:
```
console.log(pm.execution.location);
// Returns the full path of a request in array format, for example:
// ["C1", "F1", "R1"]
```

File: src/pages/docs/writing-scripts/script-references/postman-sandbox-api-reference.md (hunk 3 above)
Comment (mark-anthony-postman) [reply src/pages/docs/writing-scripts/script-references/postman-sandbox-api-reference.md:642]: I changed this to:
```
console.log(pm.execution.location.current);
// Returns the name of the current element, for example:
// F1
```

**Review by vedkribhu — APPROVED** [review]

**Pull request merged**